README.md

```diff
@@ -14,7 +14,11 @@ You can view [the application](https://wordpress.org/openverse) live on WordPres
 
 ## Technology
 
-The frontend app is built using [Vue.js](https://vuejs.org/) and [Nuxt.js](https://nuxtjs.org).
+The frontend app is built using [Vue.js](https://vuejs.org/) and [Nuxt.js](https://nuxtjs.org).To ensure a consistent code style, we use [Prettier](https://prettier.io/) and [ESLint](https://eslint.org/). We use [Jest](https://jestjs.io/) for unit testing and [Playwright](https://playwright.dev/) for end-to-end testing. We use [Storybook](https://storybook.js.org/) along with [TailwindCSS](https://tailwindcss.com/) for component development and documentation.
```

Comment:
To be fair, the Vue framework and and Nuxt metaframework are the two **_main_** technologies in use so it only makes sense to show those two. Describing every package we use in a Node app is untenable. That said I agree with your expanded selection.

Given that the number of items has been significantly increased, I think we should change this be a bullet list.

Reply:
My reasoning behind listing more packages was that most people i know search open-source softwares based on the tech stack they know/wish to exercise with. But i agree with you, listing every package we use in a node app is unrealistic. I changed the pr